Branch-SDK/Branch-SDK/Requests/BranchSetIdentityRequest.m

```diff
@@ -57,7 +57,9 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     preferenceHelper.identityID = response.data[BRANCH_RESPONSE_KEY_BRANCH_IDENTITY];
     preferenceHelper.userUrl = response.data[BRANCH_RESPONSE_KEY_USER_URL];
     preferenceHelper.userIdentity = self.userId;
-    preferenceHelper.sessionID = response.data[BRANCH_RESPONSE_KEY_SESSION_ID];
+    if (response.data[BRANCH_RESPONSE_KEY_SESSION_ID]) {
```

Comment:
We only sent down a session_id if it had changed. So this was a unique situation. (Last night to remedy this in realtime, we started sending down session_id 100% of the time.)